.github/workflows/ci.yml

```
@@ -0,0 +1,83 @@
+
+name: Server
+
+on:
+  push:
+    branches:
+      - version-13-hotfix
+      - version-13
+    paths-ignore:
+      - '**.js'
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.js'
+      - '**.md'
+
+concurrency:
+  group: version-13-landa-${{ github.event.number }}
+  cancel-in-progress: true
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    name: Server
+
+    services:
+      mariadb:
+        image: mariadb:10.3
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
```

```suggestion
          node-version: 14
```

According to the docs, frappe v13 depends on node v14

---

big yikes my bad